src/features/board/components/board-issue-drawer.tsx

```
@@ -409,6 +455,655 @@ const IssueContent = ({
+
+			<div className="space-y-2">
+				{subIssues.map((subIssue) => {
+					const isCompleted = subIssue.statusId !== parentIssue.statusId;
+					const status = statuses.find((s) => s._id === subIssue.statusId);
+
+					return (
+						<div
+							className="flex items-center gap-2 p-2 rounded-md border bg-card hover:bg-accent/50 transition-colors group cursor-pointer"
+							key={subIssue._id}
+							onClick={() => onClickIssue(subIssue)}
+						>
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)